src/VisionModel/VisionModel.jl

```diff
@@ -0,0 +1,10 @@
+module VisionModel
+
+using Flux, Images, REPL
+
+# Models
+export AlexNet
+
+include("AlexNet.jl")
```

Comment:
Another syntax guideline: file names should be lowercase unless they create a new module.